src/extra/AutoUI/index.tsx

```diff
@@ -88,6 +89,9 @@ export interface AutoUIProps<T> extends BoxProps {
 		entry: T,
 		event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
 	) => void;
+	// TODO: onChange should also be called when data in the table is sorted and when columns change
+	/** Function that gets called when filters change */
+	onChange?: (changes: { resource: string; filters?: JSONSchema[] }) => void;
```

Comment:
I would be fine even if the `resource` wasn't returned, given that the consumer is providing it anyway as part of the `model`.